packages/otel/src/index.ts

```diff
@@ -60,8 +63,11 @@ export const otel = (options: OtelOptions = {}): MiddlewareHandler => {
       },
       activeContext,
       async (span) => {
+        options.captureRequestHeaders?.push('Traceparent', 'Tracestate')
         for (const [name, value] of Object.entries(c.req.header())) {
-          span.setAttribute(ATTR_HTTP_REQUEST_HEADER(name), value)
+          if (options.captureRequestHeaders?.includes(name)) {
+            span.setAttribute(ATTR_HTTP_REQUEST_HEADER(name), value)
+          }
```

Comment:
When using `Object.entries(c.req.header())`, the `name` in L67 will always be in **lowercase**.
Therefore, it might be a good idea to convert all values in `options.captureRequestHeaders` to lowercase beforehand to ensure accurate comparison. This is also helpful because users might unintentionally input header names that are not in lowercase.